packages/appeals-service-api/src/lib/user-status.js

```diff
@@ -1,5 +1,6 @@
 const userStatus = {
-	added: 'added'
+	added: 'added',
+	confirmed: 'confirmed'
```

Comment:
might be worth coordinating with @suityou01 who is also working on these constants, but probably putting them in common package?

Reply:
Will wait on that PR to complete this one